pyro/distributions/multivariate_cauchy.py

```diff
@@ -0,0 +1,51 @@
+import torch
```

Comment:
suggestions for tests?

Reply:
add some shape tests to `tests/distributions/conftest.py`

Reply:
great done